### ISSUE TITLE
Regression: run_RF2NA.sh - R:RNA.fa No such file or directory

### ISSUE THREAD

**Issue by SuhasSrinivasan** [uw-ipd/RoseTTAFold2NA#103]
There is an issue with [run_RF2NA.sh](https://github.com/uw-ipd/RoseTTAFold2NA/blob/main/run_RF2NA.sh) which was caused due to the recent Pull Request https://github.com/uw-ipd/RoseTTAFold2NA/pull/52 

This issue occurs when running the provided example command for Protein-RNA prediction.

**The Error**
```
 -> Running command: /scratch/rf2na-new/input_prep/make_rna_msa.sh R:RNA.fa /scratch/rf2na-new/example/rna_pred RNA 18 120  
 
$ cat rna_pred/log/make_msa.RNA.stderr
 
grep: R:RNA.fa: No such file or directory
cp: cannot stat ‘R:RNA.fa’: No such file or directory
 
Error: Failed to open sequence file R:RNA.fa for reading
```

This issue did not exist before the PR.

The lines 79-82 which work
```
    type=`echo $i | awk -F: '{if (NF==1) {print "P"} else {print $1}}'`
    type=${type^^}
    fasta=`echo $i | awk -F: '{if (NF==1) {print $1} else {print $2}}'`
    tag=`basename $fasta | sed -E 's/\.fasta$|\.fas$|\.fa$//'`
```

Lines 79-84 from the PR which cause the error
```
    type=`echo $i | awk -F: '{if (NF==1) {print "P"} else {print $1}}'`
    type=${type^^}
    fasta=`echo $i`
    fasta_name=`echo $fasta | awk -F: '{if (NF==1) {print $1} else {print $2}}'`
    tag=`basename $fasta_name | sed -E 's/\.fasta$|\.fas$|\.fa$//'`
    type=`basename $type`  # extract only the last component after the last '/', so users can pass in an absolute path for each FASTA

```

This issues exists in both the `main` and `dimaio/new_config` branches.

**Comment by SuhasSrinivasan** [comment]
Submitted a Pull Request #111

**Comment by SuhasSrinivasan** [comment]
Fixed in PR #111
